src/main/java/io/mosip/mimoto/util/CredentialIssuerWellknownResponseValidator.java

```diff
@@ -17,5 +22,29 @@ public void validate(CredentialIssuerWellKnownResponse response, Validator valid
             }
             throw new InvalidWellknownResponseException(sb.toString());
         }
+
+        for(Map.Entry<String, CredentialsSupportedResponse> entry: response.getCredentialConfigurationsSupported().entrySet()){
```

Comment:
Not this way, we can try to use lambda functions map

response.getCredentialConfigurationsSupported().map(credentialSupported -> {});

Reply:
Not able to directly throw out custom exceptions from lambda, so following for loop approach to avoid complexity